File: 2 - Condições/condicoes6.php
<?php

#Escreva um programa que leia dois nomes e mostre o que contém maior quantidade de caracteres.

?>

<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Condicionais 6 - PHP</title>
  </head>
  <body>
    <?php if (!empty($_GET)): ?>
      <?php if ( strlen($_GET['nome1']) > strlen($_GET['nome2']) ): ?>
        <?= $_GET['nome1'] . " é maior."; ?>
      <?php else: ?>
        <?= $_GET['nome2'] . " é maior."; ?>
      <?php endif; ?>
    <?php else: ?>
      <form action="condicoes6.php" method="get">
        Digite um nome: <input type="text" name="nome1"/> <br>
        Digite outro nome: <input type="text" name="nome2"/> <br>
        <input type="submit" value="Enter">
      </form>
    <?php endif; ?>
  </body>
</html>

File: 2 - Condições/condicoes6.c
//Escreva um programa que leia dois nomes e mostre o que contém maior quantidade de caracteres.

#include <stdio.h>
#include <string.h>

int main() {
  char nome1[4];
  char nome2[4];
  printf("Digite dois nomes: \n");
  scanf("%s", &nome1);
  scanf("%s", &nome2);
  printf("O primeiro nome tem %d letras.\n", strlen(nome1));
  printf("O segundo nome tem %d letras.\n", strlen(nome2));
  return 0;
}
